generative_ai/function_calling.py

```diff
@@ -165,6 +165,7 @@ def generate_function_call_advanced(project_id: str) -> GenerationResponse:
         )
     )
 
+    # TODO(developer): Check latest AI models at https://cloud.google.com/vertex-ai/generative-ai/docs/model-reference/inference
```

Comment:
Let's not put the TODO here. The page where the sample is shown should have the model information.